src/sentry/static/sentry/app/components/projectHeader/projectSelector.jsx

```diff
@@ -108,10 +107,9 @@ const ProjectSelector = React.createClass({
     return label;
   },
 
-  getRawLink(project) {
+  getProjectPath(project) {
     let org = this.props.organization;
-    let urlPrefix = ConfigStore.get('urlPrefix');
-    return urlPrefix + '/' + org.slug + '/' + project.slug + '/';
+    return '/' + org.slug + '/' + project.slug + '/';
```

Comment:
untested, I assume it'll break somehow, but I didn't want to ignore it in places where its a known concern